search/requirements.txt

```diff
@@ -22,6 +23,7 @@ pytest==3.5.1
 pytest-cov==2.5.1
 pytest-mock==1.1
 pytz==2021.1
+requests-aws4auth==1.0.1
```

Comment:
Is it worth adding the aws reqs under some specific "aws" key within `extras_require`? I realize this convention hasn't been followed well for the search service but it seems like the correct approach.

Reply:
there is a follow up discussion under https://github.com/amundsen-io/amundsen/issues/1135 @sewardgw